web/gerbera.html

```diff
@@ -57,6 +57,15 @@
   })
 </script>
 
+<div class="container-fluid">
```

Comment:
Moved `warning` to top, right below menu bar.  Added `close` button on right side.